cfpb/consumerfinance.gov
# CCDB: Update dependencies

## Changes

- Updates ccdb dependencies: `core-js`, `d3`, `sinon`, `moment`, `whatwg-fetch`, and `webpack` in the main project package.json.
- Set `fetch` call on the `window`, per instructions in `whatwg-fetch`.
- Add `import 'regenerator-runtime/runtime';` in tests to satisfy babel.

## How to test this PR

1. Run `./frontend.sh`
2. Run a local server and check tools on http://localhost:8000/data-research/consumer-complaints/


## Screenshots

<img width="727" alt="Screen Shot 2020-12-11 at 1 17 52 PM" src="https://user-images.githubusercontent.com/704760/101939672-510e2b80-3bb3-11eb-9407-2b342ff584b3.png">
